### PR TITLE
Update hello.py

### PR DESCRIPTION
**Pull request by wanda-carlson**

### CODE REVIEW

File: src/hello.py [hunk 1]
```diff
@@ -2,3 +2,4 @@
 print("Wanda create a wow PR")
 print("Wanda create a wowza PRss")
 print("Wanda create a wowza PRss")
+print("Wanda create a wowza PRss")
```
Comment (henry-richard) [inline src/hello.py:5]: No

Comment (henry-richard) [inline src/hello.py:5]: Still no

**Review by henry-richard — CHANGES_REQUESTED** [review]

File: src/hello.py (hunk 1 above)
Comment (henry-richard) [inline src/hello.py:5]: But why?

Comment (wanda-carlson) [reply src/hello.py:5]: Good

Comment (henry-richard) [reply src/hello.py:5]: Okay then fine

**Review by henry-richard — APPROVED** [review]

**Pull request merged**